public/keycloak/keycloak.ts

```diff
@@ -139,8 +137,20 @@ class KeyCloakService {
     if (!isForceRefresh && (!this.kc?.tokenParsed?.exp || !this.kc.timeSkew)) {
       return
     }
+
+    let tokenholder = null;
```

Comment:
I recommend you initialize the type here.

Reply:
done